docs/changelog.rst

```diff
@@ -88,33 +120,35 @@ See :ref:`upgrade-guide-2.6` for help upgrading to the new version.
     ``CMAKE_INTERPROCEDURAL_OPTIMIZATION``, ``set(CMAKE_CXX_VISIBILITY_PRESET
     hidden)``.
 
+  * ``CUDA`` as a language is now supported.
+
   * Helper functions ``pybind11_strip``, ``pybind11_extension``,
     ``pybind11_find_import`` added, see :doc:`cmake/index`.
 
   * Optional :ref:`find-python-mode` and :ref:`nopython-mode` with CMake.
     `#2370 <https://github.com/pybind/pybind11/pull/2370>`_
 
-  * Uninstall target added.
-    `#2265 <https://github.com/pybind/pybind11/pull/2265>`_ and
-    `#2346 <https://github.com/pybind/pybind11/pull/2346>`_
+* Uninstall target added.
+  `#2265 <https://github.com/pybind/pybind11/pull/2265>`_ and
+  `#2346 <https://github.com/pybind/pybind11/pull/2346>`_
 
-* ``PYBIND11_OVERLOAD*`` macros and ``get_overload`` function replaced by
-  correctly-named ``PYBIND11_OVERRIDE*`` and ``get_override``, fixing
-  inconsistencies in the presene of a closing ``;`` in these macros.
-  ``get_type_overload`` is deprecated.
-  `#2325 <https://github.com/pybind/pybind11/pull/2325>`_
+* ``pybind11_add_module()`` now accepts an optional ``OPT_SIZE`` flag that
+  switches the binding target to size-based optimization regardless global
+  CMake build type (except in debug mode, where optimizations remain disabled).
+  This reduces binary size quite substantially (~25%).
```

Comment:
On all platforms, some platforms, certain platforms?

Reply:
Rewording this a bit, mentioning MinSizeRel (which is the correct, non-hack solution for most people, I think), and using "some platforms", since I don't think this works on absolutely every compiler and certainly was not tested everywhere.